.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "semi": false,
```

Comment:
what kind of world do we live in?

ASI doesn't exist!!;

Reply:
Haha i wrote this in 2015 when no semicolons was the way to write JS. I'd like to preserve some of that history :D